RecoLocalTracker/SiPixelRecHits/src/PixelCPETemplateReco.cc

```diff
@@ -286,14 +293,15 @@ PixelCPETemplateReco::localPosition(DetParam const & theDetParam, ClusterParam &
       //		templQbin_ );
 
 
-      float dchisq;
-      float templProbQ_;
       std::vector< SiPixelTemplateStore2D > thePixelTemp2D_;
       SiPixelTemplate2D::pushfile(ID, thePixelTemp2D_);
       SiPixelTemplate2D templ2D_(thePixelTemp2D_);
       	
-      theClusterParam.ierr =
-	SiPixelTemplateSplit::PixelTempSplit( ID, theClusterParam.cotalpha, theClusterParam.cotbeta,
+      theClusterParam.ierr = -123;
+      /*
+      float dchisq;
+      float templProbQ_;
+      SiPixelTemplateSplit::PixelTempSplit( ID, theClusterParam.cotalpha, theClusterParam.cotbeta,
```

Comment:
Hello,
May I just ask what is it all about?
This code should have nothing to do with simulations. It is used in reconstruction.
Also as far as I know we are not using the cluster splitting yet, the useSplit flag is false
by default.
Best,
Danek

On 23 Feb 2016, at 21:26, Vincenzo Innocente <notifications@github.com<mailto:notifications@github.com>> wrote:


In RecoLocalTracker/SiPixelRecHits/src/PixelCPETemplateReco.cc<https://github.com/cms-sw/cmssw/pull/13382#discussion_r53844258>:

>        std::vector< SiPixelTemplateStore2D > thePixelTemp2D_;
>        SiPixelTemplate2D::pushfile(ID, thePixelTemp2D_);
>        SiPixelTemplate2D templ2D_(thePixelTemp2D_);
>
> -      theClusterParam.ierr =
> -     SiPixelTemplateSplit::PixelTempSplit( ID, theClusterParam.cotalpha, theClusterParam.cotbeta,
> +      theClusterParam.ierr = -123;
> +      /*
> +      float dchisq;
> +      float templProbQ_;
> +      SiPixelTemplateSplit::PixelTempSplit( ID, theClusterParam.cotalpha, theClusterParam.cotbeta,


No way. Technically speaking this code belongs to Tracker DPG. Pixel in particular.
In principle I should make sure that SiPixelTemplateSplit works as before (currently will not even compile)
If you want those line to be deleted I think that tracker-DPG has to give its opinion

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/13382/files#r53844258>.